job.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+const fs = require('fs-extra');
+const _ = require('lodash');
+const DummyLogger = require('sq-logger/dummy-logger');
+const Duration = require('sq-toolkit/duration');
+
+const template = require('./lib/utils/template');
+const BigQueryError = require('./error');
+const BigQueryHelper = require('./helper');
+
+const COST_THRESHOLD_IN_GB = 100;
+
+class BigQueryJob {
+
+    /**
+     * @param {Object}    opts
+     * @param {String=}   opts.name
+     * @param {String}    opts.sqlFilename
+     * @param {Number=}   opts.costThresholdInGB
```

Comment:
Could you add a description in this jsdoc line mentioning about default value (which is COST_THRESHOLD_IN_GB)?